source/includes/_personnel.md

```diff
@@ -0,0 +1,434 @@
+
+### Request
+
+`POST https://api.handshq.com/v1/personnel`
+
+### Allowed Personnel Parameters for create
+All parameters must be nested within `personnel`
+
+Parameter | Format | Required | Description
+--------- | ------ | -------- | -----------
+first_name | String | Yes | First name of the personnel
+last_name | String | Yes | Last name of the personnel
+email | String | No | Email address of the personnel
+line_manager_id | String | No | ID of the line manager of the personnel
+role_ids | Array of strings | No | IDs of roles that exist in the training register, that the personnel holds
+
+### Response
+
+Successful requests will return a json payload of the newly created personnel and a `201` status code
+
+> 201
+
+```json
+  {
+    "data": {
+      "id": "123",
+      "type": "personnel",
+      "attributes": {
+        "first_name": "Sandra",
+        "last_name": "Smith",
+        "email": "sandra.smith@email.com",
+        "archived_at": null,
+        "type": "employee"
+      },
+      "relationships": {
+        "line_manager": {
+          "data": {
+            "id": "456",
+            "type": "line_manager"
+          }
+        },
+        "roles": {
+          "data": [
+            {
+              "id": "987",
+              "type": "role"
+            },
+            {
+              "id": "765",
+              "type": "role"
+            }
+          ]
+        }
+      }
+    }
+  }
+```
+
+## Updating a personnel
+
+```shell
+curl https://api.handshq.com/v1/personnel/[id] \
+  -H "Accept: application/json" \
+  -H "Authorization: bearer [api_token]" \
+  -H "Content-Type: application/json" \
+  --requst PATCH
+  -d '[json_payload]'
+```
+
+> Example Personnel update payload.
+
+```json
+  {
+    "personnel": {
+      "first_name": "Sally",
+      "last_name": "Smith-West",
+      "email":"sally-sw@email.com",
+      "line_manager_id": "567",
+      "role_ids": ["345"]
+    }
+  }
+```
+
+This endpoint allows you to update the first name, last name, email address, line manager and roles of an existing personnel.
+
+### Request
+
+`PATCH https://api.handshq.com/v1/personnel/[id]`
+
+
+### Allowed Personnel Parameters for update
+All parameters must be nested within `personnel`
+
+Parameter | Format | Required | Description
+--------- | ------ | -------- | -----------
+first_name | String | No | First name of the personnel
+last_name | String | No | Last name of the personnel
+email | String | No | Email address of the personnel
+line_manager_id | String | No | ID of the line manager of the personnel
+role_ids | Array of strings | No | IDs of roles that exist in the training register, that the personnel holds.
```

Comment:
@kknevitt @simhmiller should we note here that any value provided will fully override existing info, not just add to it? Am I right that that's what it does?

Reply:
Yep! I have a feeling it's mentioned in zap somewhere that this is the case but I think we need a common name to describe this behaviour (like rather than Array of strings as the data  then we could call it association_ids which has it's own section in the API describing this?